DemoApp-ObjC/DemoApp-ObjC/Ads/ALAutoLayoutBannerAdViewController.m

```diff
@@ -60,22 +60,46 @@ - (NSLayoutConstraint *)constraintWithAdView:(MAAdView *)adView andAttribute:(NS
 
 #pragma mark - MAAdDelegate Protocol
 
-- (void)didLoadAd:(MAAd *)ad {}
+- (void)didLoadAd:(MAAd *)ad
+{
+    [super logCallback: __PRETTY_FUNCTION__];
```

Comment:
`super` should only be called when overriding a function (can't think of any other use cases). These sub-classes implicitly inherit `logCallback`.

Reply:
^same goes for the rest of the `super logCallback`